examples/qwik/basic/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "qwik-basic",
+  "description": "App with Routing built-in ready to create your app",
+  "engines": {
+    "node": ">=15.0.0"
+  },
+  "private": true,
+  "trustedDependencies": [
+    "sharp"
+  ],
+  "scripts": {
+    "build": "qwik build",
+    "build.client": "vite build",
+    "build.preview": "vite build --ssr src/entry.preview.tsx",
+    "build.types": "tsc --incremental --noEmit",
+    "deploy": "echo 'Run \"npm run qwik add\" to install a server adapter'",
+    "dev": "vite --mode ssr",
+    "dev.debug": "node --inspect-brk ./node_modules/vite/bin/vite.js --mode ssr --force",
+    "fmt": "prettier --write .",
+    "fmt.check": "prettier --check .",
+    "lint": "eslint \"src/**/*.ts*\"",
+    "preview": "qwik build preview && vite preview --open",
+    "start": "vite --open --mode ssr",
+    "qwik": "qwik"
+  },
+  "devDependencies": {
+    "@builder.io/qwik": "1.2.12",
+    "@builder.io/qwik-city": "1.2.12",
+    "@types/eslint": "^8.44.7",
+    "@types/node": "^20.10.0",
+    "@typescript-eslint/eslint-plugin": "6.7.5",
+    "@typescript-eslint/parser": "6.7.5",
+    "eslint": "8.51.0",
+    "eslint-plugin-qwik": "^1.2.19",
+    "prettier": "^3.1.0",
+    "typescript": "5.1.6",
+    "undici": "^5.28.0",
+    "vite": "^4.5.0",
+    "vite-tsconfig-paths": "^4.2.1"
+  },
+  "dependencies": {
+    "@tanstack/query-core": "latest",
+    "@tanstack/qwik-query": "file:../../../packages/qwik-query"
```

Comment:
note: I think what we need to do is change this to a real dependency immediately after we've published one.